src/components/Navbar.css

```diff
@@ -9,3 +9,9 @@
 .icon {
   width: 50px;
 }
+
+.user-account {
+  width: 34px;
+  height: 50px;
+  color: #0290ff;
+}
```

Comment:
- [ ] According to the requirement you should use the exact styling please use the same color. See below for reference
- [ ] Also increase the font weight so it matches the design
### Your design
![yours](https://user-images.githubusercontent.com/81727988/216776607-3516acb3-a6bb-4b1a-a7a2-782a6a9a9312.png)

### Original design
![fimga](https://user-images.githubusercontent.com/81727988/216776618-9231860a-962d-4114-b49b-d242d5cc387f.png)